ci/scripts/build.sh

```diff
@@ -0,0 +1 @@
+make build
```

Comment:
I imagine this is too basic - when run inside CI, it'll need to look more like https://github.com/ONSdigital/dp-dataset-api/blob/develop/ci/scripts/build.sh so it is in the right directory

Same for unit.sh below